### PR TITLE
Bump prettier and ember-template-lint (devDependencies)

### PR DESCRIPTION
**Pull request by dmytro-krekota**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "none"
```
Comment (dcyriller) [inline .prettierrc:2]: May you propose stylistic changes in another PR?

Comment (dmytro-krekota) [reply .prettierrc]: I removed the "trailingComma": "none". And added commas, where the prettier wants it.

File: package.json [hunk 2]
```diff
@@ -28,11 +28,11 @@
   "license": "Apache-2.0",
   "private": false,
   "peerDependencies": {
-    "ember-template-lint": "^2.10.0",
-    "prettier": "^1.18.2"
+    "ember-template-lint": "^3.2.0",
+    "prettier": "^2.2.1"
```
[the remark this answers is not in the document]
Comment (dcyriller) [reply package.json:32]: If we apply the suggestion, we keep the compatibility with more prettier / template-lint versions.

File: .prettierrc (hunk 1 above)
Comment (dcyriller) [reply .prettierrc]: Could you explain why you want to remove the prettierrc config file and change the code style?

**Comment by dcyriller** [comment]
Thanks for your PR! We need to update the peerDependencies indeed. Left two small comments.

File: .prettierrc (hunk 1 above)
Comment (dmytro-krekota) [reply .prettierrc]: I removed it as it is "Stylistic changes"

Other things were changed automatically by "npm run format"

**Review by dcyriller — APPROVED** [review]

**Comment by dcyriller** [comment]
@dmytro-krekota we need a rebase to merge

**Comment by dmytro-krekota** [comment]
> @dmytro-krekota we need a rebase to merge
Done

**Comment by dcyriller** [comment]
@dmytro-krekota the CI is failing. As a workaround, I opened https://github.com/ember-template-lint/ember-template-lint-plugin-prettier/pull/150 with [your commit](https://github.com/ember-template-lint/ember-template-lint-plugin-prettier/pull/150/commits) to quickly fix the peerDependency issue.

Thanks for your help, the change is released in the [v2.0.1 release](https://github.com/ember-template-lint/ember-template-lint-plugin-prettier/releases/tag/v2.0.1).

**Comment by dcyriller** [comment]
I'd like to merge this one as well. Could you rebase it?

**Comment by dmytro-krekota** [comment]
Hi @dcyriller, I saw failed tests, but I need more time to investigate the problem. I will rebase and try to fix tests, sure.

File: test/unit/rules/lint-prettier-test.js [hunk 3]
```diff
@@ -80,35 +77,36 @@ test
 
 {{/my-component}}`,
       fixedTemplate: `{{#my-component}}
+
   test
+
```
Comment (dmytro-krekota) [inline test/unit/rules/lint-prettier-test.js:82]: Now prettier doesn't remove empty lines like this

**Comment by dmytro-krekota** [comment]
> I'd like to merge this one as well. Could you rebase it?

It should work now

**Review by dcyriller — APPROVED** [review]

**Comment by dcyriller** [comment]
thank you @dmytro-krekota!

**Pull request merged**